src/screens/User/components/podcastItem.tsx

```diff
@@ -1,16 +1,33 @@
 import React from 'react'
 import { View, Image, TouchableOpacity } from 'react-native'
 import { Heading, HeadingType, PlainText } from 'src/components'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { NavigationScreen } from 'src/common/enums'
+import { RootStackParamList } from 'src/common/types'
 import { Podcast } from 'src/common/types'
 import styles from './styles'
 
+type EpisodeScreenNavigationProp = StackNavigationProp<
+  RootStackParamList,
+  NavigationScreen.MY_PROFILE
+>
+
 type Props = {
   podcast: Podcast
+  navigation: EpisodeScreenNavigationProp
 }
 
-const Item: React.FC<Props> = ({ podcast }) => {
+const Item: React.FC<Props> = ({ podcast, navigation }) => {
```

Comment:
You may not to use type here. But if you want, you should use `UserScreenNavigationProp`. Because `PodcastItem` is not screen, is just a part of User screen.
```suggestion
const PodcastItem: React.FC<Props> = ({ podcast }) => {
    const navigation = useNavigation()
```